### DISCUSSION TITLE
Feature request: hdd sound / buzzer

### DISCUSSION THREAD

**Discussion by tenox7** [ZuluSCSI/ZuluSCSI-firmware#221] [Ideas]
I really miss sounds of a real hard disk, especially heads moving. It would be super cool if ZuluSCSI could have a small buzzer that could be enabled in software to make sounds along side the LED. A floppy emulator HxC has a buzzer like that and it's awesome. This would require and additional hardware. But something to consider for future version of ZuluSCSI.

**Comment by PetteriAimonen** [comment]
Related discussion: #132.

A piezo speaker can be connected to the LED header on any ZuluSCSI model, and it will make a small click on every SCSI access. Not an exact replica of actual harddrive sound, but similar.

**Comment by tenox7** [comment]
Thank you. A piezo buzzer on the LED hearder is perfectly fine. This doesnt have to be anything fancy. Just some clicky sound when there is activity.

**Comment by aperezbios** [comment]
I've converted this issue to a GitHub Discussion, as it's already implemented in recent hardware, including the [ZuluSCSI Compact Homebrew](https://store.rabbitholecomputing.com/ZuluSCSI-Compact-Homebrew-p/zuluscsi-compact-homebrew-kit.htm)
